src/main/java/com/coveo/pushapiclient/CatalogSource.java

```diff
@@ -6,7 +6,7 @@
 import java.net.http.HttpResponse;
 
 // TODO: LENS-851 - Make public when ready
-class CatalogSource implements StreamEnabledSource {
```

Comment:
i know that this should be part of future PR , but to make the sample code works , i have to make these two classes public ,